src/api/status/src/views/builds.hbs

```diff
@@ -32,6 +32,43 @@
     </div>
   </nav>
   <div class="container-fluid py-2">
+    <div id="build-header" class="border rounded p-2 mb-3 bg-white">
+      <h6 id="build-header-title" class="mb-0 text-lg text-center">
+        <i class="fa-spin fas fa-hourglass-half px-2"></i>Loading...
+      </h6>
+      <div id="build-header-info">
+        <div class="row mb-2">
+          <div class="col text-xs">Triggered on <span id="build-started"></span></div>
+          <div class="col col-md-2 text-xs">Result</div>
+          <div class="col col-md-2 text-xs">Total duration</div>
+        </div>
+        <div class="row">
+          <div class="col text-sm">
+            <div class="align-center">
+              <a id="build-sender" href="#">
+                <img
+                  class="avatar avatar-xs rounded-circle"
+                  alt=""
+                  src=""
+                  id="build-sender-img"
+                />
+                <span id="build-sender-name"></span>
+              </a>
+              synchronized
+              <span><a id="build-git-sha" href="#" class="text-decoration-underline"></a></span>
+            </div>
+          </div>
+          <div
+            id="build-result"
+            class="col col-md-2 text-dark text-sm font-weight-bolder"
```

Comment:
We can try to optimize the UI    
```    
     <div
            id="build-result"
            class="col col-md-2 text-dark text-sm font-weight-bolder fas fa-times-circle"
            style="color:red !important" 
          ></div>
 ```
![1639114960(1)](https://user-images.githubusercontent.com/55161917/145523296-25562b14-1d85-471c-b565-0d2fabfb6785.png)